### ISSUE TITLE
Request/Suggestion: Hellfire Ray

### ISSUE THREAD

**Issue by ShadyDax** [Vek17/TabletopTweaks-Base#607]
So, I kind of agree with the devs that Hellfire Ray should have a Fire descriptor - it's a fire spell, after all. But as they implemented, it receives a full bonus to damage from stuff like dragon bloodline - is it possible, maybe optionally, to make it have a Fire descriptor, but only receive half the bonuses from bloodline, just as it shows on a damage log? As of right now, in vanilla, the unholy half is equal to fire and thus receives that bonus too - so you get 0.5 of fire damage and 0.5 of unholy damage per die.

But not having it at all - as per your implementation, also sucks as it's a part of the many fire builds, and stuff like Blackened Curse - but it does not receive any bonuses as other spells do. It's a problem, because Blackened Curse then has a spell on its list that is not even an actual Fire spell. And as per pnp, Blackened Curse has other fire spells, instead of Hellfire Ray - so maybe that can be changed, too.

I'm just having a problem with Ember either casting it as a purely Evil spell without any Fire bonuses - which doesn't even fit her personality or the Curse then (as its a fire curse), and not receiving any bonuses for it - then why she even has it? Or being a completely busted spell in her hands, because of the full bonuses (on top of added Intensified Spell Metamagic).

So, it's a simple issue, but can we have an optional version of half Fire Hellfire Ray, or an optional version of Blackened Curse with other fire spell instead? Per pnp it has a Delayed Blast Fireball, which we don't even have in the game.

**Comment by CaptOcto** [comment]
Bonus damage per die are not dependent on descriptors. Changing a hellfire ray to Cold provides bonus cold damage if you have the appropriate cold dragon bloodlines and cold items (coldbite etc).

The fire descriptor affects the items Ring of Pyromania, Mallanders Folly and the fire Robe that adds +4d6. So Hellfire Ray is still really good for fire casters (but also usable for Winter Witches which I really appreciate).

**Comment by ShadyDax** [comment]
> Bonus damage per die are not dependent on descriptors. Changing a hellfire ray to Cold provides bonus cold damage if you have the appropriate cold dragon bloodlines and cold items (coldbite etc).
> 
> The fire descriptor affects the items Ring of Pyromania, Mallanders Folly and the fire Robe that adds +4d6. So Hellfire Ray is still really good for fire casters (but also usable for Winter Witches which I really appreciate).

The Fire descriptor affects bonus damage per die from Bloodlines, as well as the stuff from items you mentioned. TTT changes it to only have an Evil descriptor, so it doesn't get anything. That's the point. But Blackened Curse still gives you this spell as it is Fire, but it's not (with TTT). What are you talking about?

**Comment by ShadyDax** [comment]
Well, it's because devs clearly treat Hellfire Ray as just another Fire spell, rather than Evil, yeah. So people just ignore that it's Evil per pnp.

I wish we could fix it somehow - make it so her Curse gives her Fire Storm a bit later, maybe. Or maybe Sirocco - it's a bit lower level. Or maybe even add a spell that we lack from pnp. I don't know.

**Comment by Vek17** [comment]
I mean if it bothers you just turn off that change to Hellfire ray and you have the fire descriptor again /shrug

**Comment by ShadyDax** [comment]
Yeah and a busted ass fire spell on her hands haha that's another problem. I even mentioned it first. Oh well.

**Comment by CaptOcto** [comment]
> > Bonus damage per die are not dependent on descriptors. Changing a hellfire ray to Cold provides bonus cold damage if you have the appropriate cold dragon bloodlines and cold items (coldbite etc).
> > The fire descriptor affects the items Ring of Pyromania, Mallanders Folly and the fire Robe that adds +4d6. So Hellfire Ray is still really good for fire casters (but also usable for Winter Witches which I really appreciate).
> 
> The Fire descriptor affects bonus damage per die from Bloodlines, as well as the stuff from items you mentioned. TTT changes it to only have an Evil descriptor, so it doesn't get anything. That's the point. So what are you talking about? Have you even played with the mod? The issue is that Blackened Curse still gives you this spell as it is a Fire spell, but with TTT it's not.

Hmm super weird I was playing a Winter Witch and was certainly getting bonus damage on Hellfire Ray after converting it to cold damage from my cold bloodlines and cold boosting items. Possibly due to Frozen Caress but I didn't think it was dependent on that. If it is due to Frozen Caress then I guess the solution is to swap to a Winter Witch LOL (which has an option for a really fun Elemental Barrage build).

Overall I guess the best solution would be to remove Hellfire Ray from the list of spells available to a Blackened Cursed Oracle and give them Delayed Blast Fireball which would solve all your problems?

**Comment by Vek17** [comment]
I mean if it has the fire descriptor boosting the unholy damage is 100%
correct
